pages/home/addTask.js

```diff
@@ -0,0 +1,277 @@
+import styled from "styled-components";
+import {useState} from "react";
+import {Button} from "@mui/material";
+
+export default function AddTask() {
+  const [popup, setPopup] = useState(false);
+  const [task, setTask] = useState({
+    title: "",
+    details: "",
+    image: "",
+    whichOne: "",
+    until: "",
+    gold: 0,
+    experience: 0,
+  });
+
+  function handle(e) {
```

Comment:
Don't name the ecent `e`, name it `event`.
```suggestion
  function handle(event) {
```